airtime_mvc/application/views/scripts/embeddableplayer/embed-code.phtml

```diff
@@ -0,0 +1,242 @@
+
+                    if (data.current === null) {
+                        $("p.now_playing").html("Offline");
+                    } else {
+                        var artist = data.current.name.split(" - ")[0];
+                        var track = data.current.name.split(" - ")[1];
+                        $("p.now_playing").html(artist + "<span>" + track + "</span>");
+
+                        var current_track_end_time = new Date(data.current.ends);
+                        var current_time = new Date();
+                        //convert current_time to UTC to match the timezone of time_to_next_track_starts
+                        current_time = new Date(current_time.getTime() + current_time.getTimezoneOffset() * 60 * 1000);
+                        //TODO stop the first settimeout from executing!!
+                        time_to_next_track_starts = current_track_end_time - current_time;
+                        //console.log((time_to_next_track_starts/1000)/60);
+                        // maybe we should set time_to_next_track_starts to
+                        // (10 || 20 || etc.) minutes if its greater than that
+                        // in case of on-the-fly schedule changes
+
+
+                    }
+
+                    if (data.next === null) {
+                        $("ul.schedule_list").find("li").html("Nothing scheduled");
+                    } else {
+                        $("ul.schedule_list").find("li").html(data.next.name);
+                    }
+
+                }
+            });
+            setTimeout(getMetadata, time_to_next_track_starts);
+        }
+
+    </script>
+
+    <style type="text/css">
+        #muses_skin{width:1px; height:1px; overflow-x: hidden; overflow-y: hidden;}
+    </style>
+</head>
+
+<body>
+
+<div class="airtime_player">
+
+    <div class="airtime_header">
+        <p class="station_name">fff</p>
+    </div>
+
+    <div class="airtime_box">
+
+        <div class="airtime_button">
+            <span id="play_button" class="play_button" onclick="musesPlayer.play()"></span>
+            <span id="stop_button" class="stop_button hide_button" onclick="musesPlayer.stop()"></span>
+        </div>
+        <p class="now_playing"></p>
+
+    </div>
+
+    <!--<div class="airtime_volume">
+
+        <div class="volume_control">
+            <span class="mute"></span>
+        </div>
+
+        <div class="airtime_volume_bar">
+            <div class="airtime_volume_bar_value" style="width: 80%;"></div>
+        </div>
+
+    </div>-->
+    <div style="clear:both"></div>
+
+    <div class="airtime_schedule">
+        <p class="airtime_next">Next</p>
+        <ul class="schedule_list">
+            <li></li>
+        </ul>
+    </div>
+    <a class="airtime_pro" target="_blank" href="https://www.airtime.pro/">Powered by Airtime Pro<span class="airtime_pro_logo"></span></a>
+
+
+</div>
+
+<div id="muses_skin">
+    <script type="text/javascript">
+        var musesPlayer = new MusesPlayer();
```

Comment:
Is there any reason why this has to be here, rather than in the big script block above it?